test/components/frameTest.js

```diff
@@ -150,6 +150,44 @@ describe('clone tab', function () {
     })
   })
 
+  describe('index', function () {
+    Brave.beforeAll(this)
+
+    before(function * () {
+      this.clickWithTargetPage = Brave.server.url('click_with_target.html')
+      this.page1 = Brave.server.url('page1.html')
+
+      yield setup(this.app.client)
+      yield this.app.client
+        .waitUntilWindowLoaded()
+        .windowByUrl(Brave.browserWindowUrl)
+        .waitForUrl(Brave.newTabUrl)
+        .url(this.clickWithTargetPage)
+        .waitForVisible('#name')
+        .click('#name')
+      yield this.app.client
+        .windowByUrl(Brave.browserWindowUrl)
+        .waitForExist('.tab[data-frame-key="2"]')
+      yield this.app.client
+        .ipcSend('shortcut-set-active-frame-by-index', 0)
+        .windowByUrl(Brave.browserWindowUrl)
+        .ipcSend(messages.SHORTCUT_ACTIVE_FRAME_CLONE)
+        .windowByUrl(Brave.browserWindowUrl)
```

Comment:
this is one of the places where you really have to be careful. There is a race condition between cloning the frame and selecting the window. If the actual frame is created after the windowByUrl call, but before the waitForExist call, the waitForExist will fail because it's actually looking for the selector the new tab